auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -297,8 +297,8 @@ export default class AffiliatedEntityTable extends Mixins(DateMixin) {
   protected name (item: Business): string {
     if (this.isNumberedIncorporationApplication(item)) {
       const legalType: unknown = item.corpType.legalType
-      // *** TODO: remove fallback once namex API returns legal type?
-      return GetCorpNumberedDescription(legalType as CorpTypeCd) || 'Numbered Company'
+      // *** TODO: remove fallback once Auth API returns legal type (#14183)
+      return GetCorpNumberedDescription(legalType as CorpTypeCd) || 'Numbered Benefit Company'
```

Comment:
This will be incorrect for numbered BC/CC/ULC companies until Auth API is updated, but it will work correctly for numbered BEN companies, making this safe to merge and deploy to Prod for now.